### PR TITLE
Add opentracing.ext.logfields.

### PR DESCRIPTION
**Pull request by carlosalberto**
We were missing the Log Fields from the spec ;)

### CODE REVIEW

**Comment by carlosalberto** [comment]
Hey @yurishkuro was wondering about this - initially it's a simple change, but I remember you wanting to move `opentracing.ext.tags` to `opentracing.tags` - maybe time to do it (for both `tags` and put `logs` directly under `opentracing`?)

**Comment by yurishkuro** [comment]
Yes, let's do it - it's still backwards compatible.

I don't like the name `ext/logfields.py` though. Let's first figure out how we want the imports to look, so that they are the most convenient to use. Is there a common practice in Python for exporting constants?

**Comment by carlosalberto** [comment]
So in Python it's common to have constants under a module, which only contains constants (which is what we do now).

So what about moving `opentracing.ext.tags` to `opentracing.tags`, while exporting them in the former, to keep backwards compatibility? `opentracing.ext.tags` would look like this:

```python
# Deprecated. Use opentracing.tags instead, etc.
from ..tags import *
```

Let me know ;)

**Comment by yurishkuro** [comment]
sgtm

**Comment by carlosalberto** [comment]
Updated, and created #103 as part of putting these elements under `opentracing` directly ;)

File: opentracing/logfields.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# Copyright The OpenTracing Authors
```
Comment (yurishkuro) [inline opentracing/logfields.py]: what if we called this module "logs.py"? Then we could add helper methods like `logs.log_error(span, exc_type, exc_val, exc_tb)` (which btw is currently logged with non-standard fields: https://github.com/opentracing/opentracing-python/pull/99/files#diff-994e6f1956eaad29d5b74c46af16bf3aR217

**Comment by carlosalberto** [comment]
Hey @yurishkuro shall we merge this PR?

**Pull request merged**